modules/eco-poison-pill-about-watchdog.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/eco-poison-pill-operator.adoc
+
+:_content-type: CONCEPT
+[id="about-watchdog-devices_{context}"]
+= About watchdog devices
+
+Watchdog devices can be any of the following:
+
+* Independently powered hardware devices 
+* Hardware devices that share power with the hosts they control
+* Virtual devices implemented in software (or `softdog`)
+
+Hardware watchdog and `softdog` devices have electronic or software timers, respectively. These watchdog devices are used to ensure that the machine enters a safe state when an error condition is detected. The cluster is required to repeatedly reset the watchdog timer to prove that it is in a healthy state. This timer might elapse due to fault conditions, such as deadlocks, CPU starvation, and loss of network or disk access. If the timer expires, the watchdog device assumes that a fault has occurred and the device triggers a forced reset of the node.
+
+Hardware watchdog devices are more reliable than `softdog` devices.
+
+[id="understanding-pp-watchdog_{context}"]
+== Understanding Poison Pill Operator behavior with watchdog devices
+
+The Poison Pill Operator determines the remediation strategy based on the watchdog devices that are present.
+
+If a hardware watchdog device is configured and available, the Operator uses it for remediation. If hardware watchdog device is not configured, the Operator enables and uses a `softdog` device for remediation. 
+
+If neither watchdog devices are supported, either by the system or by the configuration, the Operator remediates nodes by using software reboot. 
```

Comment:
s/by using software reboot/rebooting the software

Reply:
@ahardin-rh - Based on https://github.com/openshift/openshift-docs/pull/40380#discussion_r805820105, can we retain the "software reboot" term?

Reply:
@abhatt-rh Based on that comment, then perhaps we can update it to be "soft reboot" instead of "software reboot"? If that's not technically correct, then we can keep the sentence as is. Thanks!

Reply:
Thank you @ahardin-rh 
I used software reboot in the context of the spec that we are documenting in the Poison Pill config YAML, which is `isSoftwareRebootEnabled`. So I'll go with it.